src/app/Dashboard/components/DashboardContainer.js

```diff
@@ -71,7 +73,7 @@ export class DashboardContainer extends Component {
         <div className='user-dashboard'>
           <h2 className='user-dashboard__name col-sm-12'>{user.name}</h2>
           <div className='col-sm-12 user-dashboard__level--container'>
-            <h3 className='user-dashboard__level'>D2</h3>
+            <h3 className='user-dashboard__level'>{dlevel.substr(0, 2)}</h3>
```

Comment:
What is happening here?

Reply:
So the backend is responding with level as this string `D1 developer`. I am getting the part `D1` only on the front end

Reply:
They should also change that to Engineer